Source/WebCore/workers/shared/SharedWorker.cpp

```diff
@@ -200,8 +200,10 @@ void SharedWorker::reportNetworkUsage(size_t bytesTransferredOverNetwork)
     ASSERT(!delta.hasOverflowed());
 
     if (delta) {
-        if (RefPtr resourceMonitor = m_resourceMonitor)
+        if (RefPtr resourceMonitor = m_resourceMonitor) {
+            RELEASE_LOG(ResourceMonitoring, "[identifier=%" PUBLIC_LOG_STRING "] SharedWorker::reportNetworkUsage to ResourceMonitor: %zu bytes", identifier().toString().utf8().data(), delta.value());
```

Comment:
Would it be more efficient to log this as an identifier, instead of converting to a string?